### ISSUE TITLE
Validation accuracy

### ISSUE THREAD

**Issue by JohannesTK** [hukkelas/DSFD-Pytorch-Inference#14]
Hi,

Thank you for the rewrite and very intuitive repo.

Have you reported validation accuracy for the RetinaNet Resnet50 and mobilenet? Or are they exactly the same as reported in here: https://github.com/biubug6/Pytorch_Retinaface#widerface-val-performance-in-single-scale-when-using-resnet50-as-backbone-net ?

Thanks,
Johannes

**Comment by hukkelas** [comment]
Sorry for the late answer.

We've not checked the validation accuracy. We did "ad-hoc" testing by checking the input/output result to the original code when we rewrote the code.
